bitmovin-analytics-datasource/src/utils/licenses.ts

```diff
@@ -0,0 +1,61 @@
+import { lastValueFrom } from 'rxjs';
+import { getBackendSrv } from '@grafana/runtime';
+import { SelectableValue } from '@grafana/data';
+
+const licenseEndpoints = [
+  {
+    endpoint: '/analytics/licenses',
+    mapperFunc: (license: any): SelectableValue => ({
```

Comment:
lets use typescript, `license` is not `any` , it is at least some structure that contains these properties
```suggestion
    mapperFunc: (license: {licenseKey: string; name: string, id: string;}): SelectableValue => ({
```
so you can extract that type to `type AnyLicense` and use that 

.. same comment for other `mapperFunc` functions

Reply:
Do you think it makes sense to install the bitmovin api sdk just for the typing? Or better to just create the AnalyticsType here?

Reply:
I would create custom Analytics type, same type having "base license fields". Such base type is not present in our api SDK, so we also did same in Dashboard, to be able to have some type in AnalyticsLicensePicker

Reply:
Done 👍